### PR TITLE
Format and mount 2nd drive

### PR DESCRIPTION
**Pull request by shaidar**
Added commands to format 2nd drive and then mount it. Also removed code that relates to creating Route53 records to Pillar data.

### CODE REVIEW

File: salt/orchestrate/logging/init.sls [hunk 1]
```diff
@@ -53,63 +51,50 @@ deploy_logging_cloud_map:
...
-    - tgt: 'P@roles:(elasticsearch|kibana|fluentd) and G@environment:operations'
+    - tgt: 'P@roles:(elasticsearch|kibana|fluentd) and G@environment:{{ ENVIRONMENT }}'
     - tgt_type: compound
     - require:
       - salt: load_pillar_data_on_logging_nodes
 
 build_logging_nodes:
   salt.state:
-    - tgt: 'P@roles:(elasticsearch|kibana|fluentd) and G@environment:operations'
+    - tgt: 'P@roles:(elasticsearch|kibana|fluentd) and G@environment:'
```
Comment (blarghmatey) [inline salt/orchestrate/logging/init.sls]: This is missing the `{{ ENVIRONMENT }}` piece after the `:`

File: salt/orchestrate/aws/logging.sls [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+  elasticsearch_index_template.present:
+    - name: 'logstash-*'
+    - definition:
+        template: 'logstash-*'
+        settings:
+          index.refresh_interval : '5s'
+        mappings:
+          _default_:
+            _all_:
+              enabled: false
```
Comment (blarghmatey) [inline salt/orchestrate/aws/logging.sls]: Do we have a particular reason for disabling the __all__ mapping, or is that just a carryover from the old map?

File: salt/orchestrate/logging/init.sls [hunk 3]
```diff
@@ -53,63 +51,57 @@ deploy_logging_cloud_map:
...
-{% do hosts.append(grains['ec2:local_ipv4']) %}
-{% endfor %}
-register_log_aggregator_internal_dns:
-  boto_route53.present:
-    - name: log-input.private.odl.mit.edu
-    - value: {{ hosts }}
-    - zone: private.odl.mit.edu.
-    - record_type: A
+# Elasticsearch Curator is used for elasticsearch snapshots
+install_elasticsearch_curator:
```
Comment (blarghmatey) [inline salt/orchestrate/logging/init.sls]: Do we have anything set up somewhere to configure curator?

File: salt/orchestrate/aws/logging.sls (hunk 2 above)
Comment (shaidar) [inline salt/orchestrate/aws/logging.sls]: Mostly a carryover from the old map, but sounds like a good idea to keep it there:
https://github.com/elastic/elasticsearch/issues/19784

File: salt/orchestrate/logging/init.sls (hunk 3 above)
Comment (shaidar) [inline salt/orchestrate/logging/init.sls]: Yeah there are two config files required in addition to running it as cron job which I just need to add to this PR

File: salt/orchestrate/logging/configure_curator.sls [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+install_elasticsearch_curator:
+  pip.installed:
+    - name: elasticsearch-curator
+    - require:
+      - salt: build_logging_nodes
+
+put_elasticsearch_snapshot_config:
+  http.query:
+    - name: http://elasticsearch.service.consul:9200/_snapshot/s3_repository/
+    - data: '
```
Comment (blarghmatey) [inline salt/orchestrate/logging/configure_curator.sls]: Rather than quoting the whole thing you could use the `|` or `>-` operators for multiline YAML.

**Comment by blarghmatey** [comment]
I think that the curator stuff fits better as an addition to the elasticsearch formula which we can then add to the top file for the Kibana node so that it gets executed in the highstate. Also, it looks like with this orchestrate configuration that the salt master would be responsible for executing the curator which I think is a better responsibility for the Kibana instance.

**Pull request closed without merging**